ajax/libs/fontawesome-iconpicker/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "fontawesome-iconpicker",
+  "version": "1.3.1",
+  "description": "Font Awesome Icon Picker plugin for Twitter Bootstrap",
+  "homepage": "https://itsjavi.com/fontawesome-iconpicker/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/itsjavi/fontawesome-iconpicker.git"
+  },
+  "filename": "js/fontawesome-iconpicker.min.js",
```

Comment:
Let's move this line between `name` and `version`.
It will helpful to web developers find the main file at the first glance.

Thank you.